### PR TITLE
Add comments component

### PR DESCRIPTION
**Pull request by dmarcoux**
It's really basic since it relies a lot on partials in OBS.

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *obs-patterns* ready!

Built with commit fd625b56e75bb7fe4ba179b4e69df9aae32c9e82

https://deploy-preview-20--obs-patterns.netlify.com

**Comment by dmarcoux** [comment]
[Document Comments Component](https://trello.com/c/L7emFWhY/865-document-comments-component)

File: _posts/components/2018-09-24-comments.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+title:  Comments
+category: components
+description: The comments section for a `Commentable` model.
+---
+
+.card.mb-3
+  %h5.card-header
+    Comments for package123 (2)
+  .card-body#comments
+    //= render partial: 'webui2/webui/comment/show', locals: { commentable: @package }
```
Comment (hennevogel) [inline _posts/components/2018-09-24-comments.html]: This is not about having a copy & paste database. You should include the code from the partial IMHO.

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: _posts/components/2018-09-24-comments.html (hunk 1 above)
Comment (dmarcoux) [reply _posts/components/2018-09-24-comments.html]: I wasn't sure how to proceed. I will keep the comment, but add the output of this partial.

Comment (dmarcoux) [reply _posts/components/2018-09-24-comments.html]: It's full of partials, so I'm not sure when to stop.

**Review by hennevogel — APPROVED** [review]

**Pull request merged**